source/administration/config-settings.rst

```diff
@@ -37,14 +39,26 @@ For any setting that is not set in ``config.json`` or in environment variables,
 .. warning::
    Database connection strings for the database read and search replicas need to be formatted using `URL encoding <https://www.w3schools.com/tags/ref_urlencode.asp>`__. Incorrectly formatted strings may cause some characters to terminate the string early, resulting in issues when the connection string is parsed.
    
-**Load Custom Configuration Defaults**
+Override Mattermost License File
+--------------------------------
+
+Starting from Mattermost v5.26, you can specify a license override using an environment variable to override any license in the database or file configuration without replacing those licenses.
+
+When starting the server, specify the license key as ``MM_LICENSE`` with the contents of a license file.
+
+.. note::
+   If ``MM_LICENSE`` is set to a non-empty string, but the license specified is not valid, the Mattermost server will be started without a license.
+   
+   In a high-availability environment, using an environment variable to override a server license only affect the individual app server and doesn't propagate to other servers in the cluster.
```

Comment:
Minor typo...   'a server license only affect<ins>**s**</ins> the ...'

Reply:
Thanks! Fixed.